fabmukunzi/posinnove
# Terms and conditions

# This PR is here for:
- Showing users which terms and conditions to follow on the site
## It has 2 pages that are both under policy route:
- Privacy and Policy
- Terms and Conditions

## How to check this?
- run `git clone https://github.com/fabmukunzi/posinnove.git`
- run `git checkout -b feat-terms-conditions`
- run `git pull origin feat-terms-conditions`
- run `yarn install`
- run `yarn dev`
- check on footer where those pages are linked
